internal/utils/main.go

```diff
@@ -30,3 +32,60 @@ func TimePointToUTCTimeStamp(providedTime xdr.TimePoint) (time.Time, error) {
 	}
 	return time.Unix(intTime, 0).UTC(), nil
 }
+
+//GetAccountAddressFromMuxedAccount takes in a muxed account and returns the address of the account
+func GetAccountAddressFromMuxedAccount(account xdr.MuxedAccount) (string, error) {
+	providedID := account.ToAccountId()
+	pointerToID := &providedID
+	return pointerToID.GetAddress()
+}
+
+//CreateSampleTx creates a transaction with a single operation (BumpSequence), the min base fee, and infinite timebounds
+func CreateSampleTx(sequence int64) xdr.TransactionEnvelope {
```

Comment:
Maybe I'm missing something; where is `CreateSampleTx` used?

Reply:
It's used in `ledger_test.go`, but I thought it would make more sense to have the function in `utils` as other tests will likely use it too.